src/store/modules/app/state.js

```diff
@@ -79,10 +79,67 @@ const state = {
     { value: 'zh-CN', label: '简体中文' }
   ],
   osKeyboardLayouts: [
+    'keymap_belgian',
```

Comment:
ok this is long enough now, please extract it to it's own JS file and import it. :)

Reply:
Done in commit ffcfc24. Although I'm not sure what's the best directory to put in or what case mode to use for the filename (`exclusion_list.js`, `longFormKeycodes.js`, and `potato-facts.js` all use a different case mode so I don't know what's the desired style for filenames)

Reply:
Ideally, we wouldn't even need to maintain a list of OS keyboard layout identifiers and just tell the configurator to list all `src/i18n/keymap_extras/keymap_*.js` files but I don't know what would be an idiomatic way to achieve that.

Reply:
> Ideally, we wouldn't even need to maintain a list of OS keyboard layout identifiers and just tell the configurator to list all `src/i18n/keymap_extras/keymap_*.js` files (and strip the path stem and .js extension) but I don't know what would be an idiomatic way to achieve that.

We would have to do it at build time and write some node js that run as part of the build and generated the file.

Reply:
> Done in commit [ffcfc24](https://github.com/qmk/qmk_configurator/commit/ffcfc2481f51d27ae6cbf2de2ac91462dfaf39c2). Although I'm not sure what's the best directory to put in or what case mode to use for the filename (`exclusion_list.js`, `longFormKeycodes.js`, and `potato-facts.js` all use a different case mode so I don't know what's the desired style for filenames)

if you want to rename all the files to one of the styles for consistency I'd be open to that. I think I'd prefer the snake casing if we were going to pick a style.